discord/app_commands/namespace.py

```diff
@@ -70,6 +70,11 @@ class Namespace:
     key-pair mapping. These attributes can be accessed using dot notation. For example, an option
     with the name of ``example`` can be accessed using ``ns.example``. If an attribute is not found,
     then ``None`` is returned rather than an attribute error.
+    
+    .. note::
```

Comment:
```suggestion

    .. warning::
```

This is failing black due to the whitespace here. Also, I think a warning is more appropriate than a note.

Reply:
Ah, already wondered why it failed